docs/sources/reference/commandline/cli.md

```diff
@@ -444,6 +449,17 @@ Currently supported options are:
     > daemon with a supported environment.
 
 ### Docker execdriver option
+Currently supported options of `zfs`:
```

Comment:
@Mic92 of course i just realized that this should be in the storagedriver section, not execdriver....